pennylane_lightning/core/src/simulators/lightning_kokkos/gates/tests/Test_StateVectorKokkos_Generator.cpp

```diff
@@ -31,14 +35,15 @@
 
 /// @cond DEV
 namespace {
+using namespace Pennylane::Gates;
 using namespace Pennylane::LightningKokkos;
 using namespace Pennylane::Util;
 using std::size_t;
 } // namespace
 /// @endcond
 
-TEMPLATE_TEST_CASE("StateVectorKokkos::applyGenerator",
-                   "[StateVectorKokkosManaged_Generator]", float, double) {
+TEMPLATE_TEST_CASE("StateVectorKokkos::applyGenerator - errors",
```

Comment:
Why the "errors" label here?

Reply:
I had to rename the test to avoid a name conflict. Because the test is only checking for a `LightningException` and I wanted to call the generic test `StateVectorKokkos::applyGenerator`, which was previously called `StateVectorKokkos::applyGeneratorXXX` with `XXX` the gate name, I renamed it with `errors`.